### PR TITLE
Adding support for Azure in SSIS referenced package template.

### PR DESCRIPTION
**Pull request by twerthi**
---

### Step template checklist

- [X] `Id` should be a **GUID** that is not `00000000-0000-0000-0000-000000000000`
  - **NOTE** If you are modifying an existing step template, please make sure that you **do not** modify the `Id` property *(updating the `Id` will break the Library sync functionality in Octopus)*. 
- [X] `Version` should be incremented, otherwise the integration with Octopus won't update the step template correctly
- [X] Parameter names should not start with `$`
- [X] **To minimize the risk of step template parameters clashing with other variables in a project that uses the step template, ensure that you prefix your parameter names (e.g. an abbreviated name for the step template or the category of the step template**
- [X] `LastModifiedBy` field must be present, and (_optionally_) updated with the correct author
- [ ] If a new `Category` has been created:
   - [ ] An image with the name `{categoryname}.png` must be present under the `step-templates/logos` folder
   - [ ] The `switch` in the `humanize` function in [`gulpfile.babel.js`](https://github.com/OctopusDeploy/Library/blob/master/gulpfile.babel.js#L92) must have a `case` statement corresponding to it

Fixes # . _If there is an open issue that this PR fixes add it here, otherwise just remove this line_

### CODE REVIEW

File: step-templates/ssis-deploy-ispac-from-referenced-package.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
     "Id": "0c8167e9-49fe-4f2a-a007-df5ef2e63fac",
-    "Name": "Deploy ispac SSIS project from Referenced Package",
-    "Description": "This step template will deploy SSIS ispac projects to SQL Server Integration Services Catalog.  The template uses a referenced package and is Worker compatible.",
+    "Name": "Deploy ispac SSIS project from a package Copy",
```
Comment (hnrkndrssn) [inline step-templates/ssis-deploy-ispac-from-referenced-package.json]: Did you mean to change the name of the template?

Comment (twerthi) [reply step-templates/ssis-deploy-ispac-from-referenced-package.json]: No ... it's a Shawn fail.  I'll get that fixed.

Comment (twerthi) [reply step-templates/ssis-deploy-ispac-from-referenced-package.json]: That has been fixed.

**Pull request merged**